site/content/en/docs/user/kwokctl-snapshot/yaml-snapshot.demo

```diff
@@ -0,0 +1,29 @@
+# Let's getting started with kwokctl!
+kwokctl create cluster
+
+# Create an nodes.
```

Comment:
You should use "an" when the first character of a word begins with a vowel. Since the word "nodes" begins with the consonant "n", then "a" should come before it. 
Also, the word "nodes" is a plural, it means many, so it becomes "node".
So, the sentence becomes `# Create a node.`

But since you are attempting to create more than one node, using "a" means singular, so you should use a plural like "the", which signifies **many**.
```suggestion
# Create the nodes.
```